### PR TITLE
Changes to update FAQ and known issues section

### PR DESCRIPTION
**Pull request by anwara000**
Changes to update FAQ and known issues section

Please use content from this document for FAQ section

https://microsoft-my.sharepoint-df.com/:w:/p/anwara/EZafBUGq44BJvCYe0-YwiJAB8y6PR1DGQaGIb0gpeoA2DA?e=sjfeoW

### CODE REVIEW

**Comment by anwara000** [comment]
@brdesai  FYI

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/customer-key-tenant-level.md [hunk 1]
```diff
@@ -420,6 +420,11 @@ For information about the availability key, see [Learn about the availability ke
 
 For information about rotating or rolling keys used with Customer Key, see [Roll or rotate a Customer Key or an availability key](customer-key-availability-key-roll.md). When you update the DEP to use the new version of the keys, you'll run the Set-M365DataAtRestEncryptionPolicy cmdlet as described earlier in this article.
 
+## Known Issues
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-tenant-level.md:423]:
```suggestion
## Known issues
```

File: microsoft-365/compliance/customer-key-tenant-level.md [hunk 2]
```diff
@@ -420,6 +420,11 @@ For information about the availability key, see [Learn about the availability ke
...
+- We see that when Customer Key is enabled, creation of team from Microsoft Teams fails.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-tenant-level.md]:
```suggestion
- When Customer Key is enabled, the creation of a new team in Microsoft Teams fails.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - I'll do a complete copy edit once the requested FAQs are included in this topic.

**Comment by yogkumgit** [comment]
@anwara-ms

**Comment by anwara000** [comment]
Hi, is it ready to be published? Can you send me preview on how the FAQ section will look like?

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@yogkumgit - I do not have access to the reference document to include the FAQs in this PR so someone else has to incorporate it.

**Comment by yogkumgit** [comment]
@KCCross

**Comment by anwara000** [comment]
for your action @KCCross  @LauraWi . Please help publish this.

**Comment by KCCross** [comment]
Several of these are incorporated through another PR. Some of these are still in discussion. Leaving this PR open while we discuss offline.

**Comment by KCCross** [comment]
Content added through other PRs. closing.

**Pull request closed without merging**